WordPress/Classes/Stores/RemoteFeatureFlagStore.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+import WordPressKit
+
+class RemoteFeatureFlagStore {
+
+    init() {
+        DDLogInfo("🚩 Remote Feature Flag Device ID: \(deviceID)")
+    }
+
+    /**
+     Fetches remote feature flags from the server.
+
+     - Parameter callback: An optional callback that can be used to update UI following the fetch. It is not called on the UI thread.
+    */
```

Comment:
nitpick. Let's go more swift-y with [comments](https://github.com/raywenderlich/swift-style-guide/#comments) instead of C-based

```suggestion
    /// Fetches remote feature flags from the server.
    ///
    /// - Parameter callback: An optional callback that can be used to update UI following the fetch. It is not called on the UI thread.
```

Reply:
Addressed in e481fa72713c06912e97e7274baf60300992f72d. I didn't use the commit suggestion just because I wanted to make the line break and indentation consistent with the other documented method in this class.